resources/content/schema/user/user-auth.json

```diff
@@ -264,6 +264,7 @@
         "project.members" : "cr",
         "project.swarm" : "cru",
         "project.kubernetes" : "cru",
+        "project.publicdns" : "cru",
```

Comment:
There needs to be a change somewhere in `test_authorization.py` to test this new field.